renderer/src/components/FilAddressForm.tsx

```diff
@@ -1,57 +1,85 @@
+import { FC, ChangeEvent, useState } from 'react'
 import { checkAddressString } from '@glif/filecoin-address'
-import { FormEvent, useEffect, useState } from 'react'
+
+interface ValidationErrorProps {
+  message: string | undefined
+}
+
+const ValidationError: FC<ValidationErrorProps> = ({ message }) => {
+  return (
+    message
+      ? <span className="err-msg" title="validation error">{message}</span>
+      : <span className='mb-[0.625rem]'>&nbsp;</span>
+  )
+}
 
 interface FilAddressFormProps {
-    address?: string
-    setAddress: React.Dispatch<React.SetStateAction<string | undefined>>
-    onValidAddress: (address: string) => void
+  setFilAddress: (address: string | undefined) => void
 }
 
-export default function FilAddressForm (props: FilAddressFormProps) {
-  const backendAddress = props.address ?? ''
-  const [address, setAddress] = useState(backendAddress)
+const FilAddressForm: FC<FilAddressFormProps> = ({ setFilAddress }) => {
   const [validationError, setValidationError] = useState<string | undefined>()
+  const [inputAddr, setInputAddr] = useState<string>('')
 
   const validateAddress = () => {
     setValidationError(undefined)
-    if (!address) return true
     try {
-      checkAddressString(address)
+      checkAddressString(inputAddr)
       return true
     } catch (err) {
       setValidationError(err instanceof Error ? err.message : '' + err)
       return false
     }
   }
```

Comment:
I think that showing the validation error might be too technical and it should be wrapped into a proper message. However, I'm not sure about the details we want to provide to the user. 
Maybe moving forward with a generic message like _"Invalid Filecoin address"_? Thoughts @juliangruber @bajtos ?

Reply:
What about `"Invalid wallet address"` or `"Invalid Filecoin wallet address"`? Iiuc it's a wallet address in the first place

Reply:
I have mixed feelings. On one hand, I agree that most users don't care about a detailed technical explanation. On the other hand, sometimes the technical explanation could be helpful - e.g. when the address has incorrect length, perhaps because the user omitted some of the characters.

I am proposing to show a generic message followed with a question-mark icon `(?)`, and when the user clicks or hovers on the icon, then show additional error details.

What do you think?

I am happy to consider different design & interaction model. The idea is to show a generic message and then allow the user to get more details if they want to.

As for the message, I think that `"Invalid Filecoin wallet address"` is very accurate but also rather long. I am voting for `"Invalid wallet address"`.

Reply:
Exposing the underlying error message in addition is a good idea, I've opened an issue for this so we can add it in an iteration: https://github.com/filecoin-project/filecoin-station/issues/145